sdks/java/extensions/euphoria/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/client/operator/base/Builders.java

```diff
@@ -186,10 +186,22 @@ default WindowedOutputBuilderT accumulatingFiredPanes() {
     /**
      * Finalizes the operator and retrieves its output dataset.
      *
-     * @param outputHints output dataset description
+     * @param outputHint output dataset description
+     * @param outputHints other output dataset descriptions
      * @return the dataset representing the new operator's output
+     * @deprecated Use {@link #output()} instead.
      */
-    PCollection<T> output(OutputHint... outputHints);
+    @Deprecated
+    default PCollection<T> output(OutputHint outputHint, OutputHint... outputHints) {
```

Comment:
Can we add a log here giving warnings that this call does effectively nothing now and will be removed?